File: python_backend/ranges/jumps/get_init_jumps.py


def get_init_jumps(offsets, board, sqr, color):
    """jumps are square that can be reached through a piece jumping"""
    jumps = []
    for offset in offsets:
        x1, y1 = offset
        jumps.append((sqr[0] + x1, sqr[1] + y1))
    return jumps

File: app (1)/src/game_logic/ranges/jumps/getInitJumps.js
export function getInitJumps(offsets, sqr) {
    /*jumps are square that can be reached through a piece jumping**/
    var x, y;
    let jumps = []
    for (var offset of offsets) {
        [x, y] = offset
        jumps.push([sqr[0] + x, sqr[1] + y])
    }
    return jumps;
}

// module.exports = getInitJumps;